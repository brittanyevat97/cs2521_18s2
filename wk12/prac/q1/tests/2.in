30 -10 5 22 71 55 

