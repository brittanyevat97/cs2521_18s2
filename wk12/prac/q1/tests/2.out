Inserted Keys:  30,  -10,  5,  22,  71,  55, 
countEven is: 3 
